pkg/network/helper.go

```diff
@@ -0,0 +1,129 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	cfTypes "github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	ColorReset  = "\033[0m"
+	ColorRed    = "\033[31m"
+	ColorGreen  = "\033[32m"
+	ColorYellow = "\033[33m"
+	ColorBlue   = "\033[34m"
```

Comment:
I believe it would be appropriate to move them, but at the moment other parts of ROSA dont really use it. It could be done as part of a separate ticket, to apply the colours to more logs and to move them to a separate file.

Reply:
sounds good to me, id like to make a UX ticket at some point on the new team doing this, I think it would be great for listing clusters for example